src/models/PartialFileFieldSubmission.php

```diff
@@ -71,4 +71,12 @@ public function canDelete($member = null)
 
         return parent::canDelete($member);
     }
+
+    /**
+     * @return string
+     */
+    public function getFilename()
+    {
+        return $this->UploadedFileID ? $this->UploadedFile()->Name : '';
```

Comment:
Is this necessary? I thought the parent of PartialFileFieldSubmission already took care of this?

Reply:
Filename returns "Uploads/file.pdf", I just want the name.